modular_zubbers/master_files/code/datums/id_trim/jobs.dm

```diff
@@ -0,0 +1,46 @@
+//New honorifics
+
+/datum/id_trim/job/blueshield/New()
+	job = /datum/job/blueshield
+	honorifics = list("Agent")
```

Comment:
only the overrides that are adding onto existing honorific lists need to be in a `New()` call, if the job has no other honorifics normally then you can just add as a regular var